tests/test_validation.py

```diff
@@ -182,3 +182,76 @@ def test_invalid_nested_object_property(app):
     assert resp.json() == {
         "error": "Invalid object type: <class 'str'>",
     }
+
+
+def test_missing_report_object_property(app):
+    app.authorize([
+        'spinta_report_insert',
+        'spinta_report_getone',
+    ])
+
+    resp = app.post('/report', json={
+        'type': 'report',
+        'report_type': 'simple',
+        'status': 'valid',
+        'count': '42',
+        'valid_from_date': '2019-04-20',
+        'update_time': '2019-04-20 03:14:15',
+        'notes': [{
+            'note': 'hello report',  # missing object properties
```

Comment:
I think note is in schema.

```yaml
  notes: {
      type: array,
      items: {
          type: object,
          properties: {
              note: {type: string},
              note_type: {type: string}
          }
      }
  }
```

Reply:
Sorry, ignore this comment.

Reply:
Yes, and test does not throw any error that `note_type` is missing, though as mentioned above `FIXME` is added to load missing values as `NA`.